xpk.py

```diff
@@ -1114,7 +1360,130 @@ def run_gke_cluster_create_command(args) -> int:
   return 0
 
 
-def create_cluster_configmap(args, system):
+def set_up_cluster_network(args) -> int:
+  """Set up GKE Cluster networks, subnets and firewall rules.
+
+  Args:
+    args: user provided arguments for running the command.
+
+  Returns:
+    0 if successful and 1 otherwise.
+  """
+  for i in range (1, 5):
+    return_code = create_cluster_network(args, i)
+    if return_code != 0:
+      return 1
+    return_code = create_cluster_subnet(args, i)
+    if return_code != 0:
+      return 1
+    return_code = create_cluster_firewall_rule(args, i)
+    if return_code != 0:
+      return 1
+  return 0
+
+
+def create_cluster_network(args, index) -> int:
+  """Create one GKE Cluster network.
+
+  Args:
+    args: user provided arguments for running the command.
+    index: index number for the network to be created.
+
+  Returns:
+    0 if successful and 1 otherwise.
+  """
+  command = (
+    f'gcloud compute --project={args.project}'
```

Comment:
`gcloud compute networks create` command will fail if the network is already created.

Can we check if it already exists and skip if so? LMK if have another suggestion!